package.json

```diff
@@ -17,28 +17,41 @@
     "lint"
   ],
   "dependencies": {
-    "@nuxtjs/google-analytics": "^2.0.2",
+    "@nuxtjs/axios": "^5.3.6",
+    "@nuxtjs/google-analytics": "^2.2.0",
     "axios": "^0.18.0",
```

Comment:
Should `axios` still be a direct dependency? It's a dependency of [@nuxtjs/axios](https://www.npmjs.com/package/@nuxtjs/axios), so unless explicitly used in the project somewhere, this dependency should be removed.

```suggestion
```

Reply:
... seems to work just fine if I remove it.